e2e/travis.sh

```diff
@@ -5,6 +5,10 @@ set -ue
 
 source ./e2e/util.sh
 
+if [ "${CONTAINER_RUNTIME}" != "docker" ]; then
+    sudo systemctl stop docker
```

Comment:
Need to stop docker because of upstream issue: https://github.com/kubernetes/kubeadm/issues/1495